### ISSUE TITLE
Enable logout from OIDC Provider

### ISSUE THREAD

**Issue by jennifer-richards** [ietf-tools/rpc#136]
The logout button ends the session with the RPC tool, but does not end the OIDC Provider's session. This means that, e.g., attempting to log out and then log back in as a different user is not possible without first manually logging out of the OP.

The datatracker is capable of RP-initiated logout as described here:
https://django-oidc-provider.readthedocs.io/en/master/sections/sessionmanagement.html

This should be available at least as an option when logging out from the RPC tool.

**Comment by jennifer-richards** [comment]
@holloway this might be something to look at alongside the login work.

This will make it a lot easier to log out and switch users while doing testing - without it, you have to mess around with private sessions or manually delete cookies to do that.

**Comment by jennifer-richards** [comment]
The OIDC side of OP logout is implemented by #291. This still needs UI.

The UI should include a checkout to make the OP logout optional - it will (I believe) impact the session used for other services authenticating via auth.ietf.org. That will require an adjustment to the backend to support the option.

**Comment by rjsparks** [comment]
OP logout literally means "logout of the datatracker" and yes, it will affect all other services using the datatracker as the identity provider.

**Comment by jennifer-richards** [comment]
Correct. It won't affect the datatracker itself, though, because it's a separate cookie for the separate domain.
